packages/cli/lib/utils.js

```diff
@@ -471,6 +471,7 @@ exports.getDataSourceConnectorName = function(datasourcesDir, dataSourceClass) {
     jsonFileContent = JSON.parse(fs.readFileSync(datasourceJSONFile, 'utf8'));
   } catch (err) {
     debug(`Error reading file ${datasourceJSONFile}: ${err.message}`);
+    err.message = `Cannot load ${datasourceJSONFile}: ${err.message}`;
```

Comment:
Personally I would like to have a new Error that wraps the original one. It's easy to do so in Java, such as `new MyError(anotherError)`, but not in JavaScript :-(.

Reply:
Something like https://javascript.info/custom-errors#wrapping-exceptions. Maybe we should introduce a helper in LB4.

Reply:
My first attempt to provide an error message that included the filename of the datasource where the failure originated, was indeed to throw a new Error.  However, @bajtos pointed out that we'd lose the stack trace by doing that -- hence the modification to the original Error's message.

Reply:
I'm fine with your fix for now.

Reply:
> Personally I would like to have a new Error that wraps the original one. It's easy to do so in Java, such as `new MyError(anotherError)`, but not in JavaScript :-(.

Have you considered using [verror](https://www.npmjs.com/package/verror)?